### ISSUE TITLE
toggle verbosity of validation set results

### ISSUE THREAD

**Issue by epwalsh** [epwalsh/pytorch-crf#23]
Only display macro scores when `verbose=False`
